### PR TITLE
Descriptions to www-o-o and edits

### PR DESCRIPTION
**Pull request by gonyere**
Added descriptions for Images and Containers and for Development collaboration. Also made a few (mostly minor) edits to most of the other descriptions for readability/grammar/etc.

### CODE REVIEW

File: _data/locales/en.yml [hunk 1]
```diff
@@ -43,7 +45,16 @@ technologies:
         url: https://github.com/openSUSE/cavil
   container:
     name: Images and Containers
-    description:
+    description: |
```
Comment (hellcp) [inline _data/locales/en.yml:48]: That description doesn't really relate to "Images and Containers" category title

Comment (gonyere) [reply _data/locales/en.yml:48]: Hmm. Does this more relate to Kiwi and Docker in OBS then? I can certainly rewrite it.

Comment (hellcp) [reply _data/locales/en.yml:48]: Indeed

**Comment by gonyere** [comment]
Updated Images and Containers description.

File: index.html.1 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
+<html>
+<head>
+<title>Index of /pub/slackware-iso/slackware64-15.0-iso on ftp.slackware.com:21</title>
```
Comment (cboltz) [inline index.html.1:4]: I somehow doubt that adding this `index.html.1` was intentional ;-)

Can you please remove it again (with another PR)?

**Pull request merged**